cypress-io/cypress
# feat(component-testing): avoid browser to close between specs

To optimize running component testing in CI and locally, we avoid opening a new browser for every spec file.

## To test
- `yarn install`
- navigate to `npm/vue`
- run the following command: `yarn cy:run --spec "**/code-coverage/*.js"`
- See that the two specs are run in the same instance of the browser.

closes CT-297

@dmtrKovalenko After speaking to Brian, I chose to make the minimum of changes to runner-ct and use the changes of URL provided by the BaseProject. It has 2 advantages:
- Less code to write because no listener to implement
- It could be used in e2e if they wish to with changes needed to runner

I hope you like it.
